### ISSUE TITLE
File Manager Shortcut

### ISSUE THREAD

**Issue by victories** [webmin/webmin#472]
Hello,

Can you open the file manager by clicking "Home Directory" in the "Edit Virtual Server" section?

Best regards,
Hasan.

![filemanagerlink](https://cloud.githubusercontent.com/assets/1463038/21296601/419f9e84-c580-11e6-8236-e1329b44274e.png)

**Comment by qooob** [comment]
It would be possible technically.

Jamie, what do you think?

**Comment by victories** [comment]
If so, can we also print the external ip address next to the ip address?
Ip addresses  172.16.3.3
External Ip Adresses  123.456.789.0

Thank you.

**Comment by swelljoe** [comment]
Can you make the IP request into a separate issue, so we can track it separately?

**Comment by swelljoe** [comment]
I  like the idea of making the home directory link to file manager (if file manager is present and permissions allow its use in that directory).

**Comment by qooob** [comment]
So if there if FM is installed and permissions are OK, make a link, containing '/filemin/index.cgi?path=myPath&external=1' and I will try to figure the rest out. If you do it now, I will add it to 18.31 path for the upcoming WM.

**Comment by swelljoe** [comment]
This kinda fell on the floor, I think.

I wonder if Jamie ever saw this one. I'd chatted with Jamie about adding File Manager to the left-hand menu (and removing the website editor link, by default, so we aren't making the left menu even more chaotically huge with options), but having this link would be also very nice.

**Comment by chris001** [comment]
Definitely.  Would be nice to have both the link which opens filemin at the home dir, and next to this link an icon which opens the built-in browser based javascript shell at the home dir.

**Comment by jcameron** [comment]
A link to the file manager from the left menu will be in the 6.00 Virtualmin release (I just implemented it last week).
